src/unix/linux_like/linux/mod.rs

```diff
@@ -4464,6 +4565,19 @@ pub const HWTSTAMP_FILTER_PTP_V2_SYNC: ::c_uint = 13;
 pub const HWTSTAMP_FILTER_PTP_V2_DELAY_REQ: ::c_uint = 14;
 pub const HWTSTAMP_FILTER_NTP_ALL: ::c_uint = 15;
 
+// linux/ptp_clock.h
+pub const PTP_MAX_SAMPLES: ::c_uint = 25; // Maximum allowed offset measurement samples.
+
+pub const PTP_CLOCK_CAPS_RSV_LEN: usize = {
+    if cfg!(any(target_arch = "sparc", target_arch = "sparc64")) {
+        12
+    } else if cfg!(any(target_env = "musl", target_env = "ohos")) {
+        13
+    } else {
+        11
+    }
+};
```

Comment:
note: `cfg_if!` does not work here; the const symbol just does not show up in the C tests.

Reply:
I just removed this const and the struct that used it for now.